17 vs code

chloemurrell opened the pull request:
I have update the readme with the write up about downloading and setting up VS code. 

Closes #17

chloe-murrell committed:
Changes to Readme

chloemurrell committed:
Update README.md

chloemurrell committed:
Update README.md

chloemurrell committed:
Update README.md

Added in section for developers - setting up VS code. Will add in using VS code at a later stage.

chloemurrell committed:
Update README.md

Added in section for developers - setting up VS code. Will add in using VS code at a later stage.

jwestw commented on README.md:
````diff
@@ -30,3 +30,59 @@ The aims of the project are:
+``` 
+Then run 
+```Python 
+git clone "copied link from GitHub"
````
May as well put the actual URL in there for now. Note that it will probably change in the future.

jwestw commented on README.md:
````diff
@@ -30,3 +30,59 @@ The aims of the project are:
+```python 
+Python --version
+```
+It will return the version of python if installed. 
````
Python usually starts with upper case "P"

jwestw commented on README.md:
```diff
@@ -30,3 +30,59 @@ The aims of the project are:
+#### Extensions
+Download the Python extension in the extensions button in the side bar. 
+
+![How to download Python extension video](https://user-images.githubusercontent.com/97117990/164270232-fdb693fa-e2fc-419a-b1dd-1514c9f438ad.mov)
```
On my ONS computer these links are not working. Can we check it on your computer please before merging?

jwestw replied:
This includes all links to .mov files.

jwestw commented on README.md:
````diff
@@ -30,3 +30,59 @@ The aims of the project are:
+#### Ensure Python is installed on system
+If you are unsure if you have python installed, you can find out within the terminal using the following code. 
+```python 
+Python --version
````
Here, `python` is a command, so should be lower case.

jwestw commented on README.md:
```diff
@@ -30,3 +30,59 @@ The aims of the project are:
+
+#### Downloading VS Code
+
+You can download VS code from internet [VS code download link](https://code.visualstudio.com/download)
```
"You can download VS Code from the [VS Code download page](https://code.visualstudio.com/download). Select the correct installer for your OS and follow the instructions."

jwestw commented:
I cannot see the .mov files so consider one of the following:
- converting to gif
- converting to webm
- just doing screenshots and hosting the png file in the README

Also please speed up the video if you can.

chloe-murrell committed:
environment.yml created

chloemurrell committed:
Update README.md

chloemurrell committed:
Update README.md

chloemurrell committed:
Update README.md

jwestw commented:
I am still not able to see the movies

![image](https://user-images.githubusercontent.com/67740306/165299944-2f0120fc-ceb2-4230-b460-8d77613689e6.png)


This is on my ONS computer. 

I wonder if it's something to do with active content being blocked.

chloemurrell committed:
Update README.md

Updated to resolve comments from previous pull request.

chloemurrell committed:
Update README.md

test for video 1

chloemurrell committed:
Update README.md

test for video 1

chloemurrell committed:
Update README.md

I have updated the readme so the videos now play at normal speed and I can access them from my personal laptop. Still not able to see them on my ONS device but as the API etc also doesn't work I don't see this as a huge issue?

chloemurrell committed:
Update README.md

I have updated the readme so the videos now play at normal speed and I can access them from my personal laptop. Still not able to see them on my ONS device but as the API etc also doesn't work I don't see this as a huge issue?

chloemurrell committed:
Merge pull request #21 from jwestw/8_virtual_environment

8 virtual environment

james-westwood reviewed (approved):
Chloe and I have discussed the videos. They are not auto playing, but when they were they were running really slowly. So not autoplaying is an acceptable compromise. 

We will split the "For Developers" section out later, if/when we have a website. For now it is all on one page.

chloe-murrell committed:
changes to readme

chloe-murrell committed:
reorder developer section

chloe-murrell committed:
concluding merge

james-westwood reviewed (approved):
Approving after rebase
